modules/virt-example-snippet-nmstate-multiple-interfaces.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-snippet-nmstate-multiple-interfaces_{context}"]
+= Example snippet: Multiple interfaces in the same Policy
+
+You can create multiple interfaces in the same Policy. These interfaces can reference each other, allowing you to build and deploy a network configuration using a single Policy manifest.
+
+The following example snippet creates a bond (`bond10`) across two NICs and a Linux bridge (`br1`) that connects to the bond.
+
+[source,yaml]
+----
+...
+interfaces:
```

Comment:
I would keep the `description` field in all examples

Reply:
That's a good idea - done